### PR TITLE
195-lint-before-commit

### PR DESCRIPTION
**Pull request by juanenrisley**
With the **Husky** library we have access to git hooks from the project. Then, using the _pre-commit_ git hook, we run the lint command before commiting thanks to the **lint-staged** library.

The lint in Github Actions tests is still there because if someone has not set up the local environment correctly, lint errors will be committed anyway.

> **Warning**
> For everything to work as expected, It is needed that dependencies are installed e.g. using `npm i`.

Closes https://github.com/CirclesUBI/circles-core/issues/195

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx lint-staged --quiet
```
Comment (JacqueGM) [inline .husky/pre-commit:4]: EOF

Comment (juanenrisley) [reply .husky/pre-commit]: Done! thank you for noticing! 😌

**Review by mikozet — APPROVED** [review]

**Review by llunaCreixent — APPROVED** [review]
loving it! thanks!

**Comment by llunaCreixent** [comment]
The `npm run prepare` script only has to be executed once per project to create the files, right?

**Pull request merged**